paddle/fluid/imperative/partial_grad_engine.cc

```diff
@@ -36,6 +36,15 @@
 namespace paddle {
 namespace imperative {
 
+struct hash_pair {
```

Comment:
NIT: suggest to name struct like "HashPair". However, since Paddle already has lot of places named like this, it also OK to me if we don't change.

Reference:
https://google.github.io/styleguide/cppguide.html#Type_Names

Reply:
DONE